public/js/NeighbourImages.js

```diff
@@ -83,48 +102,83 @@ NeighbourImages.prototype.getImages = function(sample_id) {
  */
 NeighbourImages.prototype.setImages = function() {
     var self = this;
-    var $nebula0 = $('#nebula-0');
-    var $body = $('body');
-
-    console.log(self.neighbourImages);
 
     // make all img frames empty
-    $nebula0
+    this.selectors.$nebula0
         .attr('src', '//:0')
         .attr('title', 'not found');
 
-    $('.thumbnail > img')
+    this.selectors.$thumbnailChildImg
         .attr('src', '//:0')
         .attr('title', 'not found');
```

Comment:
It's there in case images can't be found (be it because they're missing from the DB for some reason, or something went wrong during the AJAX request). I'd rather the frames in the image gallery be completely blank than still have neighbouring images belonging to the last sample that was looked at.